email.go

```diff
@@ -641,6 +641,18 @@ func headerToBytes(buff io.Writer, header textproto.MIMEHeader) {
 			switch {
 			case field == "Content-Type" || field == "Content-Disposition":
 				buff.Write([]byte(subval))
+			case field == "From" || field == "To" || field == "Cc" || field == "Bcc":
+				participants := strings.Split(subval, ";")
```

Comment:
I think you meant to split by a `, ` here, right? That's currently the delimiter used to separate addresses. Without this, if someone specifies more than one address, then the `mail.ParseAddress(v)` call fails silently and nothing gets encoded.

Reply:
Yea wow i was super confused, from the rfc its `,` i thought it would be `;` all the time.